### ISSUE TITLE
markdown source code for Alternate checkboxes screenshot

### ISSUE THREAD

**Issue by claremacrae** [Akifyss/obsidian-border#323]
Hi, I'm the maintainer of the Tasks plugin, since 2.5 years ago...

I've added one-click support for a bunch of popular themes:
https://publish.obsidian.md/tasks/Reference/Status+Collections/About+Status+Collections

I really like the look of this one, and would love to be able to add support for it.

Please could you supply the Markdown source for Column 1 in this image, to enable me to do this:

https://github.com/Akifyss/obsidian-border/raw/main/img/Checkboxes.png

**Comment by Akifyss** [comment]
Hello, and thank you for your efforts. Are you referring to markdown text? If so, here they are:

```markdown
- [ ] To Do
- [/] In Progress
- [x] Done
- [-] Cancelled
- [>] Rescheduled
- [<] Scheduled 
- [!] Important
- [?] Question
- [i] Infomation
- [S] Amount
- [*] Star
- [b] Bookmark
- [“] Quote
- [n] Note
- [l] Location
- [I] Idea
- [p] Pro
- [c] Con
- [u] Up
- [d] Down
```

**Comment by claremacrae** [comment]
Brilliant. Thanks!

One-click support for this plugin now added to Tasks:

- https://github.com/obsidian-tasks-group/obsidian-tasks/pull/3036

It will be included in the next release...

**Comment by claremacrae** [comment]
Just released in [Tasks 7.9.0](https://github.com/obsidian-tasks-group/obsidian-tasks/releases/tag/7.9.0).

**Comment by Akifyss** [comment]
Thank you for your brilliant work!
